src/grammar.rustpeg

```diff
@@ -7,11 +7,12 @@ grammar -> Grammar
   { Grammar{ imports:imports, rules:rules } }
 
 rule -> Rule
-  = exported:exportflag name:identifier returns: returntype equals expression:expression semicolon? {
-      Rule{ name: name, expr: box expression, ret_type: returns, exported: exported }
+  = exported:exportflag cached:cacheflag name:identifier returns: returntype equals expression:expression semicolon? {
```

Comment:
We're eventually going to need an attribute system that allows the flags to be in any order.